### DISCUSSION TITLE
How to have `hl-todo-occur` show entire project?

### DISCUSSION THREAD

**Discussion by refaelsh** [tarsius/hl-todo#72] [Support] [answered]
Currently, `hl-todo-occur`, show only the current buffer.
How to have `hl-todo-occur` show entire project?

**Comment by tarsius** [comment, marked as answer]
You [already](https://github.com/tarsius/hl-todo/issues/71) asked that a month ago.

**Reply by refaelsh** [reply]
Damn... Do I feel stupid? Terribly sorry for wasting your time.

**Reply by tarsius** [reply]
No problem. Things are quiet enough right now. :grin:
